### ISSUE TITLE
Tabset: close buttons not visually indicated on focus, missing text equivalent (2.4.7, 1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#866]
### Expected behavior
1. Close buttons on tabs visually indicate they have focus. 
2. Close button have text value (e.g. Close tab or Close [tab label] tab)
### Actual behavior

Close buttons have no visual indication on focus. There is not text value for the close buttons. 
(tested in Mac Chrome)

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.12.0](https://github.com/blackbaud/skyux/releases/tag/v1.12.0)
